.eslintrc.js

```diff
@@ -7,14 +7,27 @@ module.exports = {
       jsx: false,
```

Comment:
Updating libraries required some changes, and some other rules were added to match other projects.